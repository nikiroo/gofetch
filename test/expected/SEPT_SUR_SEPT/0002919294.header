0150 personnalités de France lancent un "manifeste pour l%27accueil des migrants"	null/SEPT_SUR_SEPT/0002919294		70
i  [Monde] 26-09-2018 15:24
i
i    Quelque  150 personnalités, intellectuels ou artistes en France
i    ont   lancé  mercredi  un  %27manifeste  pour  l%27accueil  des
i    migrants%27 refusant toute ...
i
