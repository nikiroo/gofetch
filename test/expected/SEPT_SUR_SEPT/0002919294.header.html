<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h1><a href='0002919294.html'>150 personnalités de France lancent un "manifeste pour l%27accueil des migrants"</a></h1>
	<div class='details'>([Monde] 26-09-2018 15:24)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Quelque 150 personnalités, intellectuels ou artistes en France ont lancé mercredi un %27manifeste pour l%27accueil des migrants%27 refusant toute ...
	</div>
<hr/>
</div>
</body>
